app/views/register/beneficiaries/charityortrust/charity/CharityAddressUKView.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.register.beneficiaries.charityOrTrust.charity.routes._
+@import models.Mode
+
+@this(
+    main_template: MainTemplate,
+    formHelper: FormWithCSRF
+)
+
+@(form: Form[_], mode: Mode, draftId: String, charityName: String, index: Int)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = s"${errorPrefix(form)} ${messages("charity.ukAddress.title")}"
+) {
+
+    @formHelper(action = CharityAddressUKController.onSubmit(mode, index, draftId), 'autoComplete -> "off") {
+
+        @components.back_link()
+
+        @components.error_summary(form.errors)
+
+        <fieldset>
```

Comment:
I think it was recommended on addresses that we don't need the fieldset and legend as it's the only _thing_ that can be entered on the page.

Recommend checking with Wendy and referring to the DAC stories.